sdcm/results_analyze.py

```diff
@@ -258,7 +262,7 @@ def check_regression(self, test_id, is_gce=False):
         # send results by email
         results = dict(test_type=test_type,
                        test_id=test_id,
-                       test_version=doc['_source']['versions']['scylla-server'],
+                       test_version=test_version_info,
                        res_list=res_list)
         logger.info('Regression analysis:')
         logger.info(pp.pformat(results))
```

Comment:
it can be very big list, change log level to debug

Reply:
changed